kentikapi/internal/errors/errors.go

```diff
@@ -0,0 +1,104 @@
+package errors
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+type Code int
+
+const (
+	AuthError          Code = 1
```

Comment:
```suggestion
	AuthError          Code = iota
```
Please use iota and drop values below for simplicity, as it was pointed out earlier. See: https://go.dev/doc/effective_go#constants

Reply:
Done